Makefile

```diff
@@ -0,0 +1,133 @@
+	}
+
+.PHONY: export-to-s3
+export-to-s3: ## Bring up a sample s3-exporter service
+	@{ \
+		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
+		export AWS_DEFAULT_REGION=$(aws_default_region); \
+		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
+		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
+		export S3_BUCKET=$(s3_bucket); \
+		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
+		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
+		docker-compose up --build -d hbase-to-mongo-export-s3; \
+	}
+
+.PHONY: restart
+restart: ## Restart hbase and other services
+	@{ \
+		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
+		export AWS_DEFAULT_REGION=$(aws_default_region); \
+		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
+		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
+		export S3_BUCKET=$(s3_bucket); \
+		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
+		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
+		docker-compose restart; \
+	}
+
+.PHONY: down
+down: ## Bring down the hbase and other services
+	@{ \
+		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
+		export AWS_DEFAULT_REGION=$(aws_default_region); \
+		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
+		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
+		export S3_BUCKET=$(s3_bucket); \
+		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
+		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
+		docker-compose down; \
+	}
+
+.PHONY: destroy
+destroy: down ## Bring down the hbase and other services then delete all volumes
+	docker network prune -f
+	docker volume prune -f
+
+.PHONY: integration
+integration: up ## Run the integration tests in a Docker container
+	@{ \
+		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
+		export AWS_DEFAULT_REGION=$(aws_default_region); \
+		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
+		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
+		export S3_BUCKET=$(s3_bucket); \
+		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
+		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
+		docker-compose up --build hbase-to-mongo-export-itest; \
+	}
+
+.PHONY: hbase-shell
+hbase-shell: ## Open an Hbase shell onto the running hbase container
+	docker-compose run --rm hbase shell
+
+.PHONY: logs-file-exporter
+logs-file-exporter: ## Show the logs of the file exporter. Update follow_flag as required.
+	docker logs $(follow_flag) hbase-to-mongo-export-file
+
+.PHONY: logs-directory-exporter
+logs-directory-exporter: ## Show the logs of the directory exporter. Update follow_flag as required.
+	docker logs $(follow_flag) hbase-to-mongo-export-directory
+
+.PHONY: logs-s3-exporter
+logs-s3-exporter: ## Show the logs of the s3 exporter. Update follow_flag as required.
+	docker logs $(follow_flag) hbase-to-mongo-export-s3
+
+.PHONY: reset-all
+reset-all: destroy up logs-directory-exporter ## Destroy all, rebuild and up all, and check the export logs
```

Comment:
You will hate this but I know you'll understand. Format off all others is"

.PHONY: name
name: ## comment
           code

But this one is:

.PHONY: name
name: code ## comment

Makes me sad :(

Reply:
That's how make dependancies work, sorry  :-)